.github/workflows/run-e2e-suite.yml

```diff
@@ -21,18 +21,18 @@ jobs:
         with:
           name: ${{ inputs.package }}
       - uses: dagger/dagger-for-github@e47aba410ef9bb9ed81a4d2a97df31061e5e842e
-        if: inputs.old-arch == false
         with:
           verb: run
           args: go run ./pkg/build/e2e --package=grafana.tar.gz --suite=${{ inputs.suite }}
       - name: Set suite name
         id: set-suite-name
+        if: always()
```

Comment:
maybe silly github actions question: what's the purpose of `if: always()`? couldn't we just remove it entirely?

Reply:
ah wait i think i figured it out... is it so the steps run even if previous steps have failed?

Reply:
Yup - its to make it upload artifacts even when the previous step (the tests) fails, which is kinda the point 😅